dispatcher/protocols.py

```diff
@@ -40,6 +48,10 @@ def close(self):
         """Close the sychronous connection"""
         ...
 
+    def verify_self_check(self, message: dict[str, Any]) -> None:
+        """Verify a received self check message"""
+        ...
```

Comment:
:+1:  Nothing to add here, just want to see this is nice to see.

Needs to be added to the protocol because it is necessary to be used by `DispatcherMain`, making it a public method. I was worried that these would be confusing and I was going too far doubling-down on these (often necessitated as I went further down mypy checks).